support/configure.inc

```diff
@@ -566,7 +566,7 @@ EOF
 
   # Add build config string declaration
   cat >> "${CONFIG_H}" <<EOF
-#define DESCRAMBLER_MAX_KEYS 64
+#define DESCRAMBLER_MAX_KEYS 8
```

Comment:
Also, this is questionable for PowerVu - I would set this to 16 (at least).

Reply:
Explanation was in other (closed) pull request. 
from oscam-emu:

<pre>

#define PVU_CW_VID      0       // VIDeo
#define PVU_CW_HSD      1       // High Speed Data
#define PVU_CW_A1       2       // Audio 1
#define PVU_CW_A2       3       // Audio 2
#define PVU_CW_A3       4       // Audio 3
#define PVU_CW_A4       5       // Audio 4
#define PVU_CW_UTL      6       // UTiLity
#define PVU_CW_VBI      7       // Vertical Blanking Interval
</pre>

So a max of 8 is plenty. (I have never seen more than 5 used.)

I generally don't care about oversized arrays, but each descrambler using multipid calls tvhcsa_init for each "possible" pid/key.

<pre>

    for (i = 0; i < DESCRAMBLER_MAX_KEYS; i++) {
      tk = &dr->dr_keys[i];
      tk->key_index = 0xff;
      tk->key_interval = tk->key_initial_interval = ms2mono(interval);
      tvhcsa_init(&tk->key_csa);
      if (!multipid) break;
    }
</pre>